ncm-systemd/src/main/pan/components/systemd/schema.pan

```diff
@@ -4,12 +4,268 @@
 
 declaration template components/${project.artifactId}/schema;
 
-include 'quattor/schema';
+include 'quattor/types/component';
+include 'components/accounts/functions';
+
+# TODO: some generic types, candidates for template-library-core
+@documentation{
+    hwloc (Portable Hardware Locality, hwloc(7)) location, e.g. node:1 for NUMAnode 1
+}
+type hwloc_location = string with match(SELF, '^[\w:.]+$');
```

Comment:
i don't like like renaming tool/domain-specifc terminology. if you search the hwloc manpage for location, it's pretty clear what a `location` is.